crates/nu-std/lib/help.nu

```diff
@@ -653,33 +653,44 @@ def show-command [command: record] {
     print ""
 }
 
-# Show help on nushell commands.
+# Show help on commands.
 export def "help commands" [
     ...command: string@"nu-complete list-commands"  # the name of command to get help on
     --find (-f): string  # string to find in command names and usage
 ] {
-    let commands = ($nu.scope.commands | where not is_extern | reject is_extern | sort-by name)
+    let nu_commands = ($nu.scope.commands | sort-by name | where not is_extern | reject is_extern )
```

Comment:
does moving `sort-by name` change something here?

Reply:
i don't think so but I can move it back

Reply:
i think it would be a bit better, i guess having the sort at the end might even be faster because we only sort fewer things? :open_mouth: